Include inbound message scanner migration

The migration created for the `core.inboundMessageScanner` (`Migration033`) was not included on node startup migration list. This is a reminder for us to ensure it gets included.

This causes periodic errors whenever the scanner worker executes which look like `16:10:48.684 [ERROR] [inboundMessageScanner/PerformTask] no such column: err`.

--- a/repo/init.go
+++ b/repo/init.go
@@ -17,7 +17,7 @@ import (
 	"github.com/tyler-smith/go-bip39"
 )
 
-const RepoVersion = "32"
+const RepoVersion = "34"
 
 var log = logging.MustGetLogger("repo")
 var ErrRepoExists = errors.New("IPFS configuration file exists. Reinitializing would overwrite your keys. Use -f to force overwrite.")

--- a/repo/migration.go
+++ b/repo/migration.go
@@ -53,6 +53,7 @@ var (
 		migrations.Migration030{},
 		migrations.Migration031{},
 		migrations.Migration032{},
+		migrations.Migration033{},
 	}
 )
 

